examples/wearable/UIComponents/contents/user-inputs-elements/button/bottom-button.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
+	<title>
+		Wearable UI
+	</title>
+	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<link href="../../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
+	<script src="../../../js/circle-helper.js">
```

Comment:
Is there an indentation in the code rule?
If there is no content, how about reducing the line?

Reply:
I'm not sure about the correct indentation code rule.
However, in our project, we're using the <script> tag with linefeed.
So I think it's better keep going to current status.